src/docs/asciidoc/reference_doc/ui_api.adoc

```diff
@@ -247,10 +248,280 @@ It can be used by a template to launch some processing when loading is complete
+
 ```
-        opfab.currentCard.registerFunctionToGetUserResponse
+        const id = opfab.currentUserCard.getProcessId();
 ```
+=== Function getSelectedEntityRecipients
 
+The template can know the list of entities selected by the users as recipients of the card by calling the _opfab.currentUserCard.getSelectedEntityRecipients()_ function.  The function will return an array of entity ids.
+
+
+```
+        const recipients = opfab.currentUserCard.getSelectedEntityRecipients();
+```
+
+=== Function getSelectedEntityForInformationRecipients
+
+The template can know the list of entities selected by the users as recipients of the card by calling the _opfab.currentUserCard.getSelectedEntityForInformationRecipients()_ function.  The function will return an array of entity ids.
+
+
+```
+        const recipients = opfab.currentUserCard.getSelectedEntityForInformationRecipients();
+```
+
+=== Function getStartDate
+
+The template can know the current startDate of the card in creation or edition by calling the _opfab.currentUserCard.getStartDate()_ function. The function will return a number corresponding to the startDate as epoch date value.
+
+```
+        const startDate = opfab.currentUserCard.getStartDate();
+```
+
+=== Function getState
+
+The template can know the state of the card by calling the _opfab.currentUserCard.getState()_ function. The function will return a string corresponding to the state.
+
+```
+        const state = opfab.currentUserCard.getState();
+```
+
+=== Function getUserEntityChildCard
+
+When editing a user card, the template can get the response sent by the entity of the current user by calling the _opfab.currentUserCard.getUserEntityChildCard()_ function. The function will return the response child card sent by current user entity or null if there is no response.
+
+```
+        const card = opfab.currentUserCard.getUserEntityChildCard();
+```
+
+[WARNING]
+The method returns only one child card and is therefore not compatible with the fact that the user is in more than one activity area authorized to send the card. In this case, if there is more than one child card, only one will be returned.
+
+
+=== Function listenToEntityUsedForSendingCard
+
+The template can receive the emitter entity of the card by registering a listener function. The function will be called by OperatorFabric after loading the template and every time the card emitter changes (if the user can choose from multiple entities).
+
+```
+        opfab.currentUserCard.listenToEntityUsedForSendingCard((entityId) => {// do some stuff with the entity id})
+```
+
+=== Function registerFunctionToGetSpecificCardInformation
+
+Register the template function to call to get user card specific information. This function will be call by opfab when user clicks on the "preview" button. More explanation can be found in the <<user_cards, user card chapter>>.
```

Comment:
"This function will be call by opfab when user" -> "This function will be called by opfab when the user"